### PR TITLE
Bug data producer id missing in reports

### PR DESCRIPTION
**Pull request by manu-govind**
Fixed issue of Reports and RDLQ not using data_producer_id

### CODE REVIEW

File: pstatus-notifications-ktor/build.gradle [hunk 1]
```diff
@@ -62,6 +62,17 @@ dependencies {
     implementation "com.expediagroup:graphql-kotlin-schema-generator:6.0.0"
     implementation "io.ktor:ktor-server-netty:2.1.0"
     implementation "io.ktor:ktor-client-content-negotiation:2.1.0"
+
+    implementation("org.jeasy:easy-rules-core:4.1.0")
```
Comment (mkrystof) [inline pstatus-notifications-ktor/build.gradle:66]: I think this bug fix is mixed up with the easy-rules stuff.  Do you need to include both in this PR?

File: pstatus-report-sink-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/ReportManager.kt [hunk 2]
```diff
@@ -239,13 +248,12 @@ class ReportManager: KoinComponent {
             this.dataStreamRoute = dataStreamRoute
             this.dexIngestDateTime = dexIngestDateTime
             this.jurisdiction = jurisdiction
-            this.senderId = senderId
             this.messageMetadata = messageMetadata
             this.stageInfo= stageInfo
             this.tags = tags
             this.data = data
-            this.jurisdiction = jurisdiction
```
Comment (mkrystof) [inline pstatus-report-sink-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/ReportManager.kt:247]: The jurisdiction should not be removed.

File: pstatus-report-sink-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/ReportManager.kt [hunk 3]
```diff
@@ -239,13 +248,12 @@ class ReportManager: KoinComponent {
             this.dataStreamRoute = dataStreamRoute
             this.dexIngestDateTime = dexIngestDateTime
             this.jurisdiction = jurisdiction
-            this.senderId = senderId
```
Comment (mkrystof) [inline pstatus-report-sink-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/ReportManager.kt:242]: The senderId should not be removed.

**Review by mkrystof — CHANGES_REQUESTED** [review]
Please remove all the easy-rules stuff from this PR.

File: pstatus-report-sink-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/ReportManager.kt (hunk 3 above)
Comment (manu-govind) [reply pstatus-report-sink-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/ReportManager.kt:242]: I removed the duplicated one

File: pstatus-report-sink-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/ReportManager.kt (hunk 2 above)
Comment (manu-govind) [reply pstatus-report-sink-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/ReportManager.kt:247]: Same. Removed the duplicated one. Somehow both were added twice.

File: pstatus-notifications-ktor/build.gradle (hunk 1 above)
Comment (manu-govind) [reply pstatus-notifications-ktor/build.gradle:66]: Not sure how it got in there. Will remove it.

**Pull request closed without merging**